src/python/pants_release/start_release.py

```diff
@@ -60,8 +84,10 @@ def notes_file_name(self) -> Path:
         return Path(f"src/python/pants/notes/{self.slug}.md")
 
 
-def relevant_shas(prior: Version, release_ref: str) -> list[str]:
-    prior_tag = f"release_{prior}"
+def relevant_shas(release_ref: str) -> list[str]:
+    # infer the changes since the most recent previous release on this branch
+    prior_tag = git("describe", "--tags", "--abbrev=0", release_ref)
```

Comment:
This changes this script to not need the `--prior` argument, and instead just look for the most recent tag before the release ref.